docs/dev/tvmdbg.md

```diff
@@ -0,0 +1,105 @@
+
+3. heads
+  heads is a list of entries as the output of the graph.
+
+4. node_row_ptr
+  node_row_ptr stores the history of forward path, so you can skip constructing the entire graph in inference tasks.
+ 
+5. attrs
+  attrs can contain version numbers or similar helpful informations.
+    `storage_id` - memory slot id for each node in the storage layout
+    `dtype` - Datatype of each node (enum value)
+    `dltype` - Datatype of each node in order
+    `shape` - Shape of each node k order
+
+Example of dumped graph:
+```
+{
+  "nodes": [                                    # List of nodes
+    {
+      "op": "null",                             # operation type = null, this is a placeholder/variable/input node
+      "name": "x",                              # Name of the argument node
+      "inputs": []                              # inputs for this node, its none since this is an argument node
+    },
+    {
+      "op": "tvm_op",                           # operation type = tvm_op, this node can be executed
+      "name": "relu0",                          # Name of the node
+      "attrs": {                                # Attributes of the node
+        "flatten_data": "0",                    # Whether this data need to be flattened
+        "func_name": "fuse_l2_normalize_relu",  # Fused function name, this function is there in the lib
+        "num_inputs": "1",                      # Number of inputs for this node
+        "num_outputs": "1"                      # Number of outputs this node produces
+      },
+      "inputs": [[0, 0, 0]]                     # Position of the inputs for this operation
+    }
+  ],
+  "arg_nodes": [0],                             # Which all nodes in this are argument nodes
+  "node_row_ptr": [0, 1, 2],                    # Row indices for faster depth first search
+  "heads": [[1, 0, 0]],                         # Position of the output nodes for this operation
+  "attrs": {                                    # Attributes for the graph
+    "storage_id": ["list_int", [1, 0]],         # memory slot id for each node in the storage layout
+    "dtype": ["list_int", [0, 0]],              # Datatype of each node (enum value)
+    "dltype": ["list_str", [                    # Datatype of each node in order
+        "float32",
+        "float32"]],
+    "shape": ["list_shape", [                   # Shape of each node k order
+        [1, 3, 20, 20],
+        [1, 3, 20, 20]]]
+  }
+}
+
+```
+
+**2. Tensor dumping**
+     The tensor received after execution is in `tvm.ndarray` type. But the UX cannot read this format. So it can be transformed to numpy format using `asnumpy()`.
+More about numpy format can be read in the below link.
+[numpy-format](https://docs.scipy.org/doc/numpy/neps/npy-format.html)
+Each node in the graph will be dumped to individual files, in the dump folder. These files will be created after execution of each node.
+
```

Comment:
Add a section on how to get these data from debugger interface. This is useful for debugger UX developers